### DISCUSSION TITLE
Cpp support LmHandlerMsgDisplay.h

### DISCUSSION THREAD

**Discussion by adixmasz** [Lora-net/LoRaMac-node#1181] [Ideas]
Can I ask for:

```c
#ifdef __cplusplus
extern "C"
{
#endif`

// Source code in header

#ifdef __cplusplus
}
#endif
```

In file LmHandlerMsgDisplay.h
I must add it manualy at each library update.

**Comment by mluis1** [comment]
I have created an issue in order to not forget to apply this recommendation. #1184
